src/overlays/actors/ovl_Fishing/z_fishing.c

```diff
@@ -129,54 +136,60 @@ ActorInit Fishing_InitVars = {
     (ActorFunc)Fishing_DrawFish,
 };
 
-static f32 D_80B7A650 = 0.0f;
+static f32 sFishingStormStep = 0.0f;
 
-static u8 D_80B7A654 = 0;
+static u8 sFishingStormTimer = 0;
 
-static f32 D_80B7A658 = 0.0f;
+static f32 sFishingStormShade = 0.0f;
 
-static Vec3f D_80B7A65C = { 0.0f, 0.0f, 0.0f };
+static Vec3f sFishingStormSfxPos = { 0.0f, 0.0f, 0.0f };
 
-static f32 D_80B7A668 = 0.0f;
+static f32 sStormSFXFreqScale = 0.0f;
 
 static u8 sSinkingLureLocation = 0;
 
-static f32 D_80B7A670 = 0.0f;
+static f32 sFishCurrentCaughSize = 0.0f;
+
+static u8 sIsRodVisible = true;
 
-static u8 D_80B7A674 = true;
+static u16 sFishingCaughtSize = 0;
 
-static u16 D_80B7A678 = 0;
+static u8 sFishingCaughtTextDelay = 0;
 
-static u8 D_80B7A67C = 0;
+static s32 sFishingTimePlayed = 0;
 
-static s32 D_80B7A680 = 0;
+static s16 sOwnerUnkTimer = 0;
 
-static s16 D_80B7A684 = 0;
+typedef enum {
+/* 0x00 */ FS_OWNER_BALD,
+/* 0x01 */ FS_OWNER_CAPPED,
+/* 0x02 */ FS_OWNER_HAIR
+} FishingOwnerHair;
 
-static u8 D_80B7A688 = 0;
-static u8 D_80B7A68C = 0;
-static u8 D_80B7A690 = 0;
+static u8 sOwnerHair = FS_OWNER_BALD; 
+static u8 sOwnerHatTheft = 0;
+static u8 sOwnerHatHooked = 0;
 
-static s16 D_80B7A694 = 0;
+static s16 sRodCastState = 0;
 
 static Vec3f sFishMouthOffset = { 500.0f, 500.0f, 0.0f };
 
 static u8 D_80B7A6A4 = 0;
 
-static f32 D_80B7A6A8 = 0.0f;
+static f32 sRodBendRotY = 0.0f;
 static f32 D_80B7A6AC = 0.0f;
 static f32 D_80B7A6B0 = 0.0f;
 static f32 D_80B7A6B4 = 0.0f;
 static f32 D_80B7A6B8 = 0.0f;
 static f32 D_80B7A6BC = 0.0f;
 static f32 D_80B7A6C0 = 0.0f;
 
-static s16 D_80B7A6C4 = 0;
-static s16 D_80B7A6C8 = 0;
+static s16 sStickxCopy = 0;
+static s16 sStickYCopy = 0;
```

Comment:
-> adjStick X/Y Prev

as commented on `Input.rel`, rel.stick_x/y stored "adjusted" stick position i.e. with a "deadzone" applied
and "prev" is our typical way of indicating "previous value", typically from previous frame